Pearson-Advance/edx-sga
# PADV-420 - Allow CCX staff to grade assignments.

## Description:

This PR:
- Adds support for CCX coaches to grade assignments.
- Adds support for CCX instructors to grade assignments.

## Notes:

- Other custom changes have not been migrated because they are no longer needed.
- CI is failing, but it is not the purpose of this ticket to solve that problem.

## How to test it:

- Install the plugin (LMS and CMS).
- Create a course and add an SGA component: https://github.com/Pearson-Advance/edx-sga/tree/pearson-release/olive.main#course-authoring-in-edx-studio
- Create a new CCX course.
- Submit a test assignment with a student user.
- Go to the unit containing the SGA component with the CCX tutor or instructor role.
- Click on the "Grade Submissions" button.
- Find the assignment and grade it.
- After grading the assignment, it should not be necessary to approve the grade.
- Go to the progress page, with the student user and check if the grade is assigned to the unit containing the SGA component.


## Reviewers:
